index.js

```diff
@@ -0,0 +1,7 @@
+const News = require('./news');
+
+const news = new News;
+let today = new Date();
+
+news.fetchNewsData(today.toISOString().slice(0, 10));
+
```

Comment:
I like what you've done here, it's going to always get news for whatever date you run the server on!